Backpack-SwiftUI/Card/Classes/BPKCard.swift

```diff
@@ -43,7 +43,7 @@ public struct BPKCard<Content: View>: View {
         }
     }
 
-    private var elevation: Elevation
+    private var elevation: Elevation?
```

Comment:
Is there a specific reason to make the elevation nullable?
If the user doesn't provide an elevation, we will fallback to 'default'.

Reply:
I thought about it like if you want no elevation, this could be represented as nil, as a lack of being a shadow.

Reply:
I feel it might be better to be explicit about it.
considering:
- Not supplying = .default
- Making it nil = no shadow
- Passing in .none = no shadow

If we make it non nullable, then we fallback to default and people supply .none to hide the shadow.